### PR TITLE
Add release notes for pipeline 0.20 to 0.22

### PR DESCRIPTION
**Pull request by piyush-garg**

### CODE REVIEW

File: openshift/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# OpenShift Pipelines 1.4
+
+# Tekton Pipelines v0.22.0
+
+## Features
```
Comment (vdemeester) [inline openshift/release/release-notes.md:5]: Waw, no "new features" in 0.22.0 ? 😅

**Review by vdemeester — APPROVED** [review]

File: openshift/release/release-notes.md (hunk 1 above)
Comment (piyush-garg) [reply openshift/release/release-notes.md:5]: Yes based on this https://github.com/tektoncd/pipeline/releases/tag/v0.22.0

**Comment by piyush-garg** [comment]
@vdemeester I am not sure, if CI would pass here, because many changes happened which are not there in this release branch like

1. From master to main
2. registry URL changes in CI

File: openshift/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+N/A
+
+## Deprecation Notices
+N/A
+
+## Backwards incompatible changes
+N/A
+
+## Fixes
```
Comment (Preeticp) [inline openshift/release/release-notes.md:14]: @piyush-garg for all versions together, there are a lot of bug fixes, are all of them relevant to us and worth documenting?

File: openshift/release/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+* :bug: Set minAvailable:1 to unblock node upgrades ([#3784](https://github.com/tektoncd/pipeline/pull/3784))
+Modify webhook PodDisruptionBudget minAvailable to 1, so node upgrades aren't blocked
+
+* :bug: Losslessly roundtrip Pipelines with Finally from beta to alpha and back ([#3779](https://github.com/tektoncd/pipeline/pull/3779))
+v1beta1 Pipelines can now be requested with v1alpha1 version without losing Finally tasks. Applying the returned v1alpha1 version will store the resource as v1beta1 with the Finally section restored to its original state.
+
+* :bug: Short term fix for Cloud Event Source ([#3761](https://github.com/tektoncd/pipeline/pull/3761))
+Resolves [#2676](https://github.com/tektoncd/pipeline/issues/2676) by providing Cloud Event source value when selfLink unset
+
+## Misc
```
Comment (Preeticp) [inline openshift/release/release-notes.md:34]: So far we have never documented work in Misc sections, do we need this?

File: openshift/release/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+Fixed bug where sorted order of step statuses did not match order of step containers.
+
+* :bug: set task as failed ([#3571](https://github.com/tektoncd/pipeline/pull/3571))
+
+Declare task failure when it hits CreateContainerConfigError instead of setting it to unknown i.e. running.
+
+* :bug: Move resolution of Pipeline Results to end of completed/successful reconcile ([#3684](https://github.com/tektoncd/pipeline/pull/3684))
+
+## Misc
```
Comment (Preeticp) [inline openshift/release/release-notes.md:113]: So far we have never documented work in Misc sections, do we need this?

File: openshift/release/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+* :hammer: Remove support for build-gcs and the gcs-fetcher image ([#3771](https://github.com/tektoncd/pipeline/pull/3771))
+
+Remove support for build-gcs and the gcs-fetcher image
+
+* :hammer: Remove tekton.dev/task label from taskrun of clustertasks ([#3764](https://github.com/tektoncd/pipeline/pull/3764))
+Remove tekton.dev/task label from taskrun of clustertasks
+
+# Tekton Pipelines v0.21.0
+
+## Upgrade Notices
```
Comment (Preeticp) [inline openshift/release/release-notes.md:45]: Is this something that needs to be documented?

File: openshift/release/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+## Fixes
+
+* :hammer: [cherry-pick] validate execution status variable ([#3697](https://github.com/tektoncd/pipeline/pull/3697))
+
+Avoid validating task results while validating context variable to access execution status since it follows similar pattern $(tasks.taskname.results.status) where status is result of some task compared to context variable for referencing execution status $(tasks.taskname.status).
+
+# Tekton Pipelines v0.20.0
+
+## Upgrade Notices
```
Comment (Preeticp) [inline openshift/release/release-notes.md:137]: Is this something that needs to be documented?

**Pull request merged**